public/locales/hi/translation.json

```diff
@@ -873,7 +873,7 @@
     "dependentLocality": "आश्रित इलाका",
     "sortingCode": "कोड क्रमबद्ध करना",
     "state": "राज्य/प्रान्त",
-    "userRegistrationRequired": "उपयोगकर्ता पंजीकरण आवश्यक",
+    "Is Public": "सार्वजनिक है",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.